treemacs.el

```diff
@@ -605,6 +605,40 @@ Delete all elements whose car is ‘eq’ to KEY from ALIST."
   (beginning-of-line)
   (forward-char))
 
+(defun treemacs--path-in-dir? (path dir)
+  "Is PATH in directory DIR?"
+  (s-starts-with? (concat dir "/") path))
```

Comment:
I think it is. Suppose you have a folder "/folder/" which you delete and you also have some buffers open from "/folder-bah/". Not adding the slash would also close all the buffers from that folder as well, since dir names don't contain the last slash by default.